src/Apps/__tests__/Fixtures/Artist/Components/ArtistHeader.tsx

```diff
@@ -27,4 +29,4 @@ export const ArtistHeaderFixture = {
     cv: true,
     auction_lots: true,
   },
-}
+} as ArtistHeader_Test_QueryRawResponse["artist"]
```

Comment:
FYI It’s better to move this to the variable definition, I think TS will give you better info.

Reply:
It's actually there too! This one is unneeded 🙈

Reply:
😸